public/js/app/components/MemoTable.jsx

```diff
@@ -1,51 +1,49 @@
 import React from 'react'
+import { seasonStr } from '../util/helpers'
+import Table from './Table'
 
 const row = (translation, courseCode, language, courseMemo) => {
-  const { courseOffering, isPdf, memoName, memoVersionsAndUrls } = courseMemo
+  const { semester, courseOffering, isPdf, memoName, memoVersionsAndUrls } = courseMemo
+  return [
+    seasonStr(translation.course_short_semester, semester),
+    courseOffering,
+    <ul className="link-list">
+      {memoName && <li key={memoName}>{memoName + ':'}</li>}
+      {memoVersionsAndUrls.map((v, index) => (
+        <li key={index}>
+          {/* eslint-disable-next-line react/jsx-no-target-blank*/}
```

Comment:
I'm on a hunt for `eslint-disable-next-line` and I can't stop ;-) 
So according to this [rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md) this is to prevent vulnerabilities.

According to [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#security_and_privacy) this might not actually be a problem anymore in recent browsers.

It seems though that we already conditionally add `rel="noreferrer"` if we have a pdf, so maybe the rule is just not smart enough. Could you try removing it and see if it gives an error (on a side note - does eslint actually work in this project?)